components/datetime/benches/datetime.rs

```diff
@@ -0,0 +1,195 @@
+use criterion::{criterion_group, criterion_main, Criterion};
+use std::fmt::Write;
+
+use icu_datetime::date::DateTime;
+use icu_datetime::options::{self, DateTimeFormatOptions};
+use icu_datetime::provider::DummyDataProvider;
+use icu_datetime::DateTimeFormat;
+
+fn datetime_benches(c: &mut Criterion) {
+    let datetimes = vec![
+        DateTime::new(2001, 9, 8, 18, 46, 40),
+        DateTime::new(2017, 7, 13, 19, 40, 0),
+        DateTime::new(2020, 9, 13, 5, 26, 40),
+        DateTime::new(2021, 1, 6, 22, 13, 20),
+        DateTime::new(2021, 5, 2, 17, 0, 0),
+        DateTime::new(2021, 8, 26, 10, 46, 40),
+        DateTime::new(2021, 12, 20, 3, 33, 20),
+        DateTime::new(2022, 4, 14, 22, 20, 0),
+        DateTime::new(2022, 8, 8, 16, 6, 40),
+        DateTime::new(2033, 5, 17, 20, 33, 20),
+    ];
+    let values = &[
+        ("pl", options::style::Date::Full, options::style::Time::None),
+        ("pl", options::style::Date::Long, options::style::Time::None),
+        (
```

Comment:
Nitpick:

This is one of those cases where I hate a fixed number of columns strictly enforced :-)
It forces you to wrap a line for 2 or 3 characters, breaking consistency, and making the whole thing less readable.

If this is done manually, would you consider unwrapping some of these entries?
Or wrap them all.

Also: nitpick, you don't have to do anything, not even comment, just ignore, or implement, your call :-)